docs/guides/monitor/kubernetes-k8s-netdata.md

```diff
@@ -29,8 +29,10 @@ your k8s infrastructure.
 -   A [Helm chart](https://github.com/netdata/helmchart), which bootstraps a Netdata Agent pod on every node in your
     cluster, plus an additional parent pod for storing metrics and managing alarm notifications.
 -   A [service discovery plugin](https://github.com/netdata/agent-service-discovery), which discovers and immediately
-    monitors 22 different services that might be running inside of your cluster's pods. Service discovery happens
-    without manual intervention as pods are created, destroyed, or moved between nodes. [Compatible
+    monitors 23 specific services, plus any endpoints covered by our [generic Prometheus
```

Comment:
I think it would be better to not use exact number (23), otherwise we need to update docs after every new go collector.

Reply:
and [service discovery](https://github.com/netdata/agent-service-discovery) itself doesnt monitor services.

Reply:
@ilyam8 Good call on both fronts. Updated!